css/css-font-loading/fontfaceset-has.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head id="head">
+<meta charset="utf-8">
+<link rel="author" title="Myles C. Maxfield" href="mmaxfield@apple.com">
+<link rel="help" href="https://drafts.csswg.org/css-font-loading-3/#dom-fontfaceset-has">
+<meta name="assert" content="Ensure that calling FontFaceSet.has() works properly." />
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style id="style">
+@font-face {
+    font-family: "WebFont";
+    src: url("resources/Rochester.otf") format("opentype");
+}
+</style>
+</head>
+<body>
+<script>
+test(function(t) {
+    let fonts = document.fonts;
+    let font = fonts.keys().next().value;
+    let font2 = new FontFace("WebFont2", "url('resources/Rochester.otf') format('opentype')");
```

Comment:
Is using a font with the same name part of the test, or would it be better to distinguish them?